lib/registrant.rb

```diff
@@ -0,0 +1,20 @@
+class Registrant
+  attr_reader :name, :age, :permit
+
+  attr_accessor :license_data
+
+  def initialize(name, age, permit=false)
+    @name = name 
+    @age = age
+    @permit = permit
+    @license_data = {:written=>false, :license=>false, :renewed=>false}
```

Comment:
```suggestion
    @license_data = {written: false, license: false, renewed: false}
```
This format is slightly more conventional